### PR TITLE
fix (cronTasks): account for new date filter format , feat(translations): add french

### LINKED ISSUE #15: Publishing every time
On a linux server, it seems like the plugin is publishing not relating on the date. Even if I put a date that is far in time to publish:
image
image
When the cron action gets launched, the entry get published not considering the date.
image

I checked dates and they seem great:
image

### PR DESCRIPTION
**Pull request by GautierDele**
closes #15

### CODE REVIEW

**Comment by GautierDele** [comment]
@ComfortablyCoding considering strapi's reaction time to major bugs this could be great merging this one as quick as possible :)

I know you are a member of Strapi but I think you know about it too (see https://github.com/strapi/strapi/issues/11892 ...)

File: server/config/cron-tasks.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = {
 				const records = await getPluginService(strapi, 'actionService').find({
 					filters: {
 						executeAt: {
-							$lte: new Date(),
+							$lte: (new Date()).getTime(),
```
Comment (ComfortablyCoding) [inline server/config/cron-tasks.js:14]: This is equivalent to Date.now() which makes is the preferred method of retrieving the unix epoch.

```suggestion
							$lte: Date.now(),
```

**Review by ComfortablyCoding — CHANGES_REQUESTED** [review]
Thank you for the PR!

I have left a comment on where I believe we can improve the changes. Other than the requested change all looks good.

P.S. I am not a Strapi Employee :sweat_smile:

**Comment by GautierDele** [comment]
Oh ok there is someone called "daedalus" on their team that's why

**Review by ComfortablyCoding — APPROVED** [review]

**Comment by ComfortablyCoding** [comment]
Looks good, tyvm. It will be included in the next release.

>Oh ok there is someone called "daedalus" on their team that's why

I believe you are still referring to me, their is no daedalus on the Strapi team.

**Comment by GautierDele** [comment]
@ComfortablyCoding could we make a release only for this since it's kinda blocking the usage ?

**Comment by ComfortablyCoding** [comment]
Yes, I plan to do one today.

**Comment by ComfortablyCoding** [comment]
Shipped in v1.1.3

**Pull request merged**